_inc/client/components/product-card/action.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const ProductCardAction = ( { intro, label, onClick, primary, href } ) => (
+	<div className="product-card__action">
+		{ intro && <h4 className="product-card__action-intro">{ intro }</h4> }
+		<Button
+			className="product-card__action-button"
+			href={ href }
+			onClick={ onClick }
+			primary={ primary }
+			target="_blank"
```

Comment:
We might want to make the "open in new window" behavior optional, but since this component isn't used anywhere else, it's an overkill right now.